src/core/Akka.Tests/Actor/ActorSystemSpec.cs

```diff
@@ -131,10 +141,67 @@ public void Throw_exception_when_register_callback_after_shutdown()
             Assert.Equal("ActorSystem already terminated.", ex.Message);
         }
 
+        [Fact]
+        public void Reliably_create_waves_of_actors()
+        {
+            var timeout = Dilated(TimeSpan.FromSeconds(20));
+            var waves = Task.WhenAll(
+                Sys.ActorOf(Props.Create<Wave>()).Ask<string>(50000),
+                Sys.ActorOf(Props.Create<Wave>()).Ask<string>(50000),
+                Sys.ActorOf(Props.Create<Wave>()).Ask<string>(50000));
+
+            waves.Wait(timeout.Duration() + TimeSpan.FromSeconds(5));
+
+            Assert.Equal(new[] { "done", "done", "done" }, waves.Result);
+        }
+
+        [Fact]
+        public void Find_actors_that_just_have_been_created()
+        {
+            Sys.ActorOf(Props.Create(() => new FastActor(new TestLatch(), TestActor)).WithDispatcher("slow"));
+            Assert.Equal(typeof(LocalActorRef), ExpectMsg<Type>());
+        }
+
+        [Fact]
+        public void Reliable_deny_creation_of_actors_while_shutting_down()
```

Comment:
This test is failing, so it's either not implemented correctly or I have found a bug ;-)
You can find the scala test [here](https://github.com/akka/akka/blob/master/akka-actor-tests/src/test/scala/akka/actor/ActorSystemSpec.scala#L288)